### DISCUSSION TITLE
Where i can download Osiris.dll?

### DISCUSSION THREAD

**Discussion by Marcos666** [danielkrupinski/Osiris#3409] [Q&A] [answered]
Hi, i'm new on this web, and i actually only wan't the Osiris.dll, where i can download just that, i don't know about compile, so i can't do it from myself. Thanks. 😋

**Comment by Nep-Timeline** [comment]
there not dll file, you need to compile it yourself
To compile it you need Visual Studio and Windows SDK.

**Reply by Marcos666** [reply]
There's any video tutorial, of how to do it? I do not have any clue, of how to work.😭😅

**Reply by PEXCH-SENTXMENT** [reply]
@Marcos666 ๖ۣۣۜŚђΰvΐ#3147

**Reply by PEXCH-SENTXMENT** [reply]
discord

**Comment by wemanzoz** [comment, marked as answer]
google: how to download visual studio, then open sln, change to release, ctrl + b

**Comment by diekunstderfuge** [comment]
You can download it in https://cheater.ninja/

**Reply by Marcos666** [reply]
Thanks! 🥉

**Comment by playday3008** [comment]
If you already registered on GitHub, you can download artifacts on GitHub Actions from my fork
